<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-B0DD0C2" failures="4" tests="5" name="Test" time="17.1" errors="0" timestamp="2024-08-22T15:21:19 IST">
  <testcase classname="TestScript.ProductPageTest" name="validatePrizeSection" time="1.866">
    <failure type="java.lang.AssertionError" message="expected [https://www.saucedemo.com/inventory.html] but found [https://www.saucedemo.com/]">
      <![CDATA[java.lang.AssertionError: expected [https://www.saucedemo.com/inventory.html] but found [https://www.saucedemo.com/]
at Page.ProductPageObject.<init>(ProductPageObject.java:24)
at TestScript.ProductPageTest.validatePrizeSection(ProductPageTest.java:49)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- validatePrizeSection -->
  <testcase classname="TestScript.ProductPageTest" name="validateISProductDisplayed" time="1.659">
    <failure type="java.lang.AssertionError" message="expected [https://www.saucedemo.com/inventory.html] but found [https://www.saucedemo.com/]">
      <![CDATA[java.lang.AssertionError: expected [https://www.saucedemo.com/inventory.html] but found [https://www.saucedemo.com/]
at Page.ProductPageObject.<init>(ProductPageObject.java:24)
at TestScript.ProductPageTest.validateISProductDisplayed(ProductPageTest.java:59)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- validateISProductDisplayed -->
  <testcase classname="TestScript.ProductPageTest" name="validateAddToCart" time="1.947">
    <failure type="java.lang.AssertionError" message="expected [https://www.saucedemo.com/inventory.html] but found [https://www.saucedemo.com/]">
      <![CDATA[java.lang.AssertionError: expected [https://www.saucedemo.com/inventory.html] but found [https://www.saucedemo.com/]
at Page.ProductPageObject.<init>(ProductPageObject.java:24)
at TestScript.ProductPageTest.validateAddToCart(ProductPageTest.java:69)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- validateAddToCart -->
  <testcase classname="TestScript.ProductPageTest" name="addProductToCart" time="2.629"/>
  <testcase classname="TestScript.ProductPageTest" name="ProductPageValidation" time="7.354">
    <failure type="java.lang.AssertionError" message="expected [https://www.saucedemo.com/inventory.html] but found [https://www.saucedemo.com/]">
      <![CDATA[java.lang.AssertionError: expected [https://www.saucedemo.com/inventory.html] but found [https://www.saucedemo.com/]
at Page.ProductPageObject.<init>(ProductPageObject.java:24)
at Page.LoginPageObject.correctUserNameAndPassword(LoginPageObject.java:206)
at Page.LoginPageObject.validateLoginFunction(LoginPageObject.java:171)
at TestScript.ProductPageTest.ProductPageValidation(ProductPageTest.java:38)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- ProductPageValidation -->
</testsuite> <!-- Test -->
